pkg/info/info.go

```diff
@@ -17,6 +17,9 @@ const (
 	BACKPLANE_CONFIG_DEFAULT_FILE_PATH = ".config/backplane"
 	BACKPLANE_CONFIG_DEFAULT_FILE_NAME = "config.json"
 
+	// Session
+	BACKPLANE_SESSION_DEFAULT_FILE_PATH = "backplane"
```

Comment:
As mentioned before, can we make this configurable in `.config/backplane/config.json`?

Reply:
Done